common/uefi_scripts/startup.nsh

```diff
@@ -19,6 +19,36 @@
 echo -off
 connect -r
 
+for %a in 0 1 2 3 4 5 6 7 8 9 A B C D E F then
+        if exist FS%a:\acs_tests\parser\Parser.efi then
```

Comment:
First check config file exist ..if not dont run anything and print error message.
   then check if parser exist ..else error